kytos/core/helpers.py

```diff
@@ -99,10 +101,20 @@ def done_callback(future):
             """Done callback."""
             if not future.exception():
                 _ = future.result()
+            else:
+                exc_str = f"{type(future.exception())}: {future.exception()}"
+                args = (
+                    getattr(future, "__inner_args")
+                    if hasattr(future, "__inner_args")
+                    else tuple()
+                )
+                LOG.error(f"listen_to handler: {handler}, "
+                          f"args: {args}, exception: {exc_str}")
 
         def inner(*args):
             """Decorate the handler to run in the thread pool."""
             future = executor.submit(handler, *args)
+            setattr(future, "__inner_args", args)
```

Comment:
As you now included all args, I believe calling the attribute `__args` makes more sense.
```suggestion
                    getattr(future, "__args")
                    if hasattr(future, "__args")
                    else tuple()
                )
                LOG.error(f"listen_to handler: {handler}, "
                          f"args: {args}, exception: {exc_str}")

        def inner(*args):
            """Decorate the handler to run in the thread pool."""
            future = executor.submit(handler, *args)
            setattr(future, "__args", args)
```

Reply:
Sounds good @ajoaoff, let's go for it, I've just checked `Future`s` source code it should be all set too.